public/pages/CreateTrigger/components/TriggerQuery/TriggerQuery.js

```diff
@@ -113,17 +112,22 @@ const TriggerQuery = ({
             )}
           />
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiFormRow label="Trigger condition response:" fullWidth>
+            <EuiCodeEditor
+              theme={isDarkMode ? 'sense-dark' : 'github'}
+              width="100%"
+              height="200px"
+              value={getExecuteMessage(executeResponse)}
```

Comment:
If trigger run successfully, will show 'true' to user, can we change to a more readable message? Such as "Trigger ran successfully"

Reply:
file a separate issue to track suggestion.

https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/issues/62